gapic/schema/api.py

```diff
@@ -341,7 +341,7 @@ def disambiguate_keyword_sanitize_fname(
 
         # Parse the google.api.Service proto from the service_yaml data.
         service_yaml_config = service_pb2.Service()
-        ParseDict(opts.service_yaml_config, service_yaml_config)
+        ParseDict(opts.service_yaml_config, service_yaml_config, True)
```

Comment:
I think Python should allow named parameter here?
```suggestion
        ParseDict(opts.service_yaml_config, service_yaml_config, ignore_unknown_fields=True)
```